example-gto/code/mlem/.github/workflows/deploy-model-with-mlem.yml

```diff
@@ -0,0 +1,47 @@
+name: Deploy MLEM model after GTO Stage assignment
+on:
+  push:
+    tags:
+      - "*"
+env:
+  HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+jobs:
+  check:
```

Comment:
what does it check? can we embed into deploy, thus simplifying it?

Reply:
1. Good point, changed that. It parses a GTO git tag. 
2. It can be embedded - e.g. it's embedded in `main`. The idea here was to showcase a different pattern - how to separate parsing from deploying. This is convenient if deployment have many steps - then in each step we'll need to put `if: needs.check.outputs.event == 'assignment'` and it can become error-prone.

Reply:
Got it. I was thinking that we might not need outputs, dependencies - can be simpler for an example repo. Your call on this.